doc: document that NEARFULL conditions can result in slow writes (#42749)
ceph/ceph: doc/cephfs/troubleshooting.rst

* osdc: Dumps the current ops in-flight to OSDs (ie, file data IO)
* osdmap: Dumps the current OSDMap epoch, pools, and OSDs

> If the osdmap or pool are in a NEARFULL condition, then the kernel cephfs

Reviewer: ```suggestion
If the data pool is in a NEARFULL condition, then the kernel
```

Author: Looks good. I can't commit it via github web ui, so I'll make the change and re-push.
